packages/libs/react-ui/src/components/NavHeader/NavHeaderSelectOption.tsx

```diff
@@ -0,0 +1,13 @@
+import type { FC } from 'react';
+import React from 'react';
+
+export interface INavHeaderSelectOptionProps
+  extends Omit<React.HTMLAttributes<HTMLOptionElement>, 'as'> {
+  children: React.ReactNode;
+  value?: string[] | string | number;
+}
+
+export const NavHeaderSelectOption: FC<INavHeaderSelectOptionProps> = ({
```

Comment:
Do you think we need to expose this? The other Select component doesn't export the `Option` component even though it has one (which we should either remove or export), but right now we aren't adding anything to the options. What do you think @KristinaSpasevska @timoheddes

Reply:
I think it makes sense to match the other Select component and thus not export it.

Reply:
Same goes for me, if that's the way the Select was built, this should be similar